os_migrate/tests/unit/test_workload.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
I wanted to comment we should have a functional export test but then realized GH Actions won't let us create an instance. So we'll only test this in end-to-end tests eventually (cc FYI @ccamacho ).